src/client.ts

```diff
@@ -294,13 +296,16 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
           success: false,
           reason: 'TIMEOUT',
           message:
-            'failed to initialize onReady before timeout, either the datafile or user info was not set before the timeout',
+            'failed to initialize onReady before timeout, either the datafile, user info or qualified segments was not set before the timeout',
```

Comment:
I believe this timeout is covering datafile ready timeout, not covering "qualifed segments" failure. The original message makes more sense.